src/taskpane/components/ReferenceList.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
I guess this comes from the fact that the react components are not correctly resolved as being used, right? Wouldn't this be better fixed by some kind of react plugin for eslint?

Reply:
If my memory serves me this was my proposed solution to get rid of,

```
/home/runner/work/JabRef-Word-Addin/JabRef-Word-Addin/src/taskpane/components/ReferenceList.tsx
Error:   23:5  error  'ev' is defined but never used       no-unused-vars
Error:   24:5  error  'checked' is defined but never used  no-unused-vars
```

in https://github.com/JabRef/JabRef-Word-Addin/runs/3080393821. But perhaps the problem was that the GitHub action wasn't using the current PRs `.eslintrc.json`?

The "origin" of the issue should be using `no-unused-vars` instead of `@typescript-eslint/no-unused-vars`, but I can't replicate it locally.